rdt/transformers/utils.py

```diff
@@ -42,7 +42,7 @@ def _max_repeat(options, max_repeat):
         max_ = max_repeat
 
     option, args = options[0]
-    generator, size = _GENERATORS[option](args, max_repeat)
+    generator, size = _GENERATORS[option](args, max_repeat)  # pylint: disable=W0612
```

Comment:
Pylint started complaining about this, however `generator` is being used in the `for for loop` this gets raised anyways...